.github/workflows/python-package.yml

```diff
@@ -11,21 +11,45 @@ on:
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
+    name: Test Python ${{ matrix.python-version }} (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest]
         python-version: ["3.8", "3.9", "3.10", "3.11"]
+        include:
+          - os: windows-latest
+            python-version: "3.8"
+          - os: macOS-latest
+            python-version: "3.8"
```

Comment:
Hi @s-weigand , here are the jobs that I'm seeing in the pipeline:

<img width="412" alt="image" src="https://github.com/jsh9/pydoclint/assets/25124332/6ada29ae-3cdf-42d5-a0dd-7f15b5d107f4">

This means we are only running the checks on Windows and macOS on Python 3.8.  Is there a reason why we don't run Windows and macOS checks on Python 3.9, 3.10, and 3.11?

Reply:
Just to save some CI time, since those tests are mostly for testing OS related issues, but they could also be activated.

Reply:
I think it's probably worth spending a bit more CI time just to make sure we've cover all our bases.  Let me change it to all 4 Python versions and all 3 operating systems then.